### ISSUE TITLE
[BUG] Cannot Log in 

### ISSUE THREAD

**Issue by AlexyGalaxy** [charlesbel/Microsoft-Rewards-Farmer#162]
I am not sure what the issue is or if I am the only one getting this error.

Start script. Goes to log in.
>Login Error 

Does not enter email or password just gives error.

**Comment by AlexyGalaxy** [comment]
Fixed the error. Updated code is on my github. https://github.com/AlexyGalaxy/Microsoft-Rewards-Farmer

**Comment by DoShhPlease** [comment]
> Fixed the error. Updated code is on my github. https://github.com/AlexyGalaxy/Microsoft-Rewards-Farmer

Great job, works almost flawlessly, thanks a lot! 

One small "issue", unfortunately since I'm a beginner in python I can't really debug a program such as this, i got a bunch of errors before it actually completed "More promotions". If you need more info somehow then let me know - I'm happy to provide it.
![Screenshot 2023-03-06 181232](https://user-images.githubusercontent.com/74171904/223195873-ba3b6bfb-1cbd-4b34-8406-cab0e4ed58fa.png)

**Comment by AlexyGalaxy** [comment]
first you need to set up your err logger, instructions are in the README.

Disable headless mode and watch the script run. Once you find what is not running correctly take a picture of the browser window and post it here.

**Comment by DoShhPlease** [comment]
Well.. I ran it again with another account with the same settings just to confirm that the errors arent isolated to one account or first run of the python file for any reason and indeed the same 12 errors occured. So, I did as you said: disabling headless mode and setting up error logger then running it again. No errors this time, across both accounts. Seems like something about doing at least one of those two fixed the errors from occuring? Not sure if any of that info can help your debugging but thanks anyway. :)

**Comment by AlexyGalaxy** [comment]
So it ran without errors?

**Comment by DoShhPlease** [comment]
Indeed it did, new plot twist though. Ran again and we got the error to replicate with the 2 settings changed. There's not much to screenshot, all i see is that firstly, it would open the more promotions quiz page but not answer any of the questions, then it would close that page and try again (re-open more promotions quiz page but not answer the question, then close the tab). after some iterations of that, it reloads the main microsoft rewards dashboard page over and over, while simultaneously printing the above shown errors after each reload (as in the same reload as pressing the F5 key)
